### PR TITLE
Release v1.12.0

### PR DESCRIPTION
**Pull request by Amir-P**

### CODE REVIEW

**Comment by Amir-P** [comment]
I had to remove `publish_to` because of pub.dev hence had to remove local Parchment too. @amantoux

File: packages/parchment/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.12.0
+* Introduce AutoFormats to handle automatice text formatting and make Heuristics only responsible for validity of document
```
Comment (amantoux) [inline packages/parchment/CHANGELOG.md:2]: automatic (spelling)

File: packages/fleather/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 1.12.0
+
+* Introduce AutoFormats to handle automatice text formatting and make Heuristics only responsible for validity of document
```
Comment (amantoux) [inline packages/fleather/CHANGELOG.md:3]: automatic (spelling)

**Review by amantoux — CHANGES_REQUESTED** [review]
Small spelling issue

**Review by amantoux — APPROVED** [review]
LGTM

**Pull request merged**